### PR TITLE
Array less than or equal migration

### PR DESCRIPTION
**Pull request by viatk**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayLessThanOrEqualOperator.java [hunk 1]
```diff
@@ -70,4 +70,39 @@ public static boolean lessThanOrEqual(
 
         return leftArray.getPositionCount() <= rightArray.getPositionCount();
     }
+    @TypeParameter("E")
```
Comment (cberner) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayLessThanOrEqualOperator.java]: There should be a space after methods and before the start of the next one

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayLessThanOrEqualOperator.java [hunk 2]
```diff
@@ -13,51 +13,36 @@
...
-import static com.facebook.presto.metadata.Signature.orderableTypeParameter;
 import static com.facebook.presto.spi.StandardErrorCode.INTERNAL_ERROR;
-import static com.facebook.presto.spi.type.BooleanType.BOOLEAN;
 import static com.facebook.presto.spi.type.TypeUtils.readNativeValue;
 import static com.facebook.presto.type.ArrayType.ARRAY_NULL_ELEMENT_MSG;
 import static com.facebook.presto.type.TypeUtils.checkElementNotNull;
-import static com.facebook.presto.util.Reflection.methodHandle;
 
+@ScalarOperator(LESS_THAN_OR_EQUAL)
+@Description("Compares two arrays if one is less than or equal then the other")
```
Comment (cberner) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayLessThanOrEqualOperator.java]: Operators don't need a description

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayLessThanOrEqualOperator.java [hunk 3]
```diff
@@ -13,51 +13,36 @@
...
 public class ArrayLessThanOrEqualOperator
```
Comment (cberner) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayLessThanOrEqualOperator.java]: This should be `final`, if your IntelliJ doesn't provide a warning, ask David how yours should be setup

**Comment by cberner** [comment]
Merged

**Comment by viatk** [comment]
@cberner, im confused by your 'Merged' comment - do you want me to address the comments and reopen the PR or is it all merged already?

**Comment by cberner** [comment]
Nope. I merged it :) I just commented so you would know for the future

**Pull request closed without merging**